### DISCUSSION TITLE
Add FTP server

### DISCUSSION THREAD

**Discussion by Gunter1710** [jomjol/AI-on-the-edge-device#647] [Ideas]
Hi, 
I would like to have an FTP server for the SDcard. It is very practical that you can view files via "File Server", but a download of e.g. the images or whole folders is not possible. Some files are also not displayed, such as /img_tmp/alg_roi.jpg.

**Comment by jomjol** [comment]
Hi,
if someone would like to implement and it still fits into the remaining free flash memory (12%), I would give it a change. I myself have no time to do so and don't see the direct benefit.
Regarding the not displayed files: they are simply not written to the sd card anymore, but generated "on the fly" out of the memory to limit the sd card access, which is a limiting factor to stability.
